profiles/base.yaml

```diff
@@ -191,9 +191,8 @@ crossing_training_requires_movement:
 - Thievery
 - Theurgy
 - Trading
-forage_item:
+forage_item: rock
```

Comment:
Could you also remove the default case from `crossing-training` (line 585)? (`|| 'rock'`)